### PR TITLE
Core: Feature - Web socket map update

### PR DESCRIPTION
**Pull request by cjmconie**
This PR adds the `'map'` web socket event to the WebSocketManager and GeoPosRegion.

### CODE REVIEW

File: BlockV/Core/Web Socket/Models/WSMapEvent.swift [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+//
+
+import Foundation
+import MapKit
+
+/*
+ {
+  "msg_type": "map",
+  "payload": {
+    "event_id": "map_ed949f70-d520-4c3a-8f78-0ada05678160",
```
Comment (MalcolmnDEV) [inline BlockV/Core/Web Socket/Models/WSMapEvent.swift]: Should these be public?

File: Example/BLOCKv Unit Tests/Models/Web Socket/MockWebSocket.swift [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+import Foundation
+
+struct MockWebSocket {
+    
+    static let mapEvent = """
+    {
+      "msg_type": "map",
+      "payload": {
+        "event_id": "map_83ce7912-82ea-45c4-9b91-36daa3e61234",
```
Comment (MalcolmnDEV) [inline Example/BLOCKv Unit Tests/Models/Web Socket/MockWebSocket.swift]: Is this dummy data ?

Comment (cjmconie) [reply Example/BLOCKv Unit Tests/Models/Web Socket/MockWebSocket.swift]: Yes it is.

File: BlockV/Core/Web Socket/Models/WSMapEvent.swift (hunk 1 above)
Comment (cjmconie) [reply BlockV/Core/Web Socket/Models/WSMapEvent.swift]: Removed

**Pull request merged**